cmd/maya-exporter/app/collector/mock/mock.go

```diff
@@ -0,0 +1,43 @@
+package mock
```

Comment:
Can we have `pkg/prometheus/exporter/mock/v1alpha1` as the package

Reply:
Sounds good